foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/config/SystemProperties.java

```diff
@@ -75,6 +77,20 @@
      */
     public static final String ENFORCE_TARGET_SERVER = "eclipselink.target-server.enforce";
 
+    /**
+     * This system property can be set the specific time zone used by ConversionManager to convert
+     * LocalDateTime, OffsetDateTime, and OffsetTime types.
+     */
+    public static final String CONVERSION_USE_TIMEZONE = "org.eclipse.persistence.conversion.useTimeZone";
```

Comment:
I'd prefer to stick with existing naming convention used by other properties - use `eclipselink._whatever_` instead of `org.eclipse.persistence._whatever_`

Reply:
This is a port of #801 to master, this should have been raised at that time.